### ISSUE TITLE
Improvements to GoutPal Search Engine

### ISSUE THREAD

**Issue by kct2020** [kct2020/goutpal-com-skeleventy#14]
## Improvements to GoutPal Search Engine Summary
- Page Affected by this Issue: [GoutPal Search Engine](https://cse.google.com/cse?cof=FORID:0&cx=partner-pub-4857169685716700:9780732506)
- Purpose of Issue: E.g. to improve usability of GoutPal Search Engine.
- Progress Notes [ [explanation](https://github.com/kct2020/goutpal-com-skeleventy/issues/9) ] : None. 
- Related Pages/Issues: None.
- Status: Waiting for comments from people who want a better search engine. For example, you might want to see some search tips. So please share any questions, experiences, or opinions about using the GoutPal Search Engine.

What are your experiences using the GoutPal search engine? Please explain what you were looking for and how good the results were. Then we can decide if search engine tips will be useful. Or consider other useful search features.

**If you have any contributions to make to improving search features, add your comments here. Comments here should address specific points covered by, or missing from, [GoutPal Search Engine](https://cse.google.com/cse?cof=FORID:0&cx=partner-pub-4857169685716700:9780732506). Otherwise, use [GoutPal Discussions](https://github.com/kct2020/goutpal-com-skeleventy/discussions) for chats with fellow gout sufferers.**

![goutpal-badge-330x65](https://user-images.githubusercontent.com/58899793/145753854-513c44a0-412f-4c26-9c63-5330270f63ad.png)
.

**Comment by kct2020** [comment]
I've added a link to [GoutPal Search Engine](https://cse.google.com/cse?cof=FORID:0&cx=partner-pub-4857169685716700:9780732506) here and above. In order to clarify what this Issue is about.

Also, I've marked this as "Waiting Comment" because I really need to know your thoughts about search results before I proceed. But you don't have to restrict your comments to GoutPal Search Engine results. Because I'm interested in learning as much as I can about how you use searching to improve the management of your gout.
